schema/schema.sql

```diff
@@ -0,0 +1,79 @@
+CREATE DATABASE catalog;
+
+-- Create item table
+CREATE TABLE item (
+  id INT GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
+  genre_id INT,
+  author_id INT,
+  label_id INT,
+  publish_date DATE,
+  archived BOOLEAN,
+  FOREIGN KEY (genre_id) REFERENCES genre(id),
+  FOREIGN KEY (author_id) REFERENCES author(id),
+  FOREIGN KEY (label_id) REFERENCES label(id)
+);
+
+-- Create genres table
+CREATE TABLE Genres (
+  id INT Generated ALWAYS AS IDENTITY PRIMARY KEY,
+  name VARCHAR(100),
+  foreign key (item_id) references item(id)
+);
+
```

Comment:
- [ ] @Naledi-Dikgale , @Patfarmurs, and @citec-47 Great work implementing the changes requested, however, you forgot to add an items array in your Genres table to match the database UML gave as previously requested, this also applies to the Author and Label tables.Kindly consider implementing theses changes.
![image](https://github.com/Naledi-Dikgale/catalog-of-my-things/assets/61408860/b7017874-7184-4018-a065-35ee21274348)